client/src/ui/ToasterWebview.ts

```diff
@@ -46,6 +46,16 @@ export class ToasterPanel {
     ToasterPanel.currentPanel = new ToasterPanel(panel)
   }
 
+  public static async startToasterInBrowser (): Promise<void> {
+    await ToasterPanel.runToasterCommand('source toaster start', true)
```

Comment:
Getting this error on my cqfd config:
```
zsh:source:1: no such file or directory: oe-init-build-env
 *  Executing script: source toaster start
 *  Bitbake process failed with code 127
 *  Terminal will be reused by BitBake, press any key to close it.
```
Well it's not supposed to work anyway on zsh and cqfd. But you can't assume oe-init-build-env is in the current directory.

Reply:
Simply supplying the toaster commands wouldn't work. And there is nothing about zsh or cqfd in the ticket description.